webapp/server.js

```diff
@@ -68,7 +74,16 @@ useHttps = useHttps.toLowerCase();
 
 // Production session data
 const session = require('express-session');
-const AzureTablesStoreFactory = require('connect-azuretables')(session);
+var redis = require("redis");
+var redisStore = connect_redis(session);
+var redisClient = redis.createClient({
+    // these need to be pulled from env.
+    // Redis elasticache needs to be in terraform (see beacons)
+    
+    
+    host: 'localhost', 
```

Comment:
This needs to be pulled in from "process". So we have these ones in our local .env, but the real ones in the prod .env etc